packages/bot-web-ui/src/components/trade-animation.jsx

```diff
@@ -63,6 +96,9 @@ TradeAnimation.propTypes = {
 export default connect(({ run_panel, contract_card }) => ({
```

Comment:
Don't forget to add propTypes for the newly added props.

Reply:
thanks 👍